frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -233,15 +233,19 @@ export const ExpressionWidget = <Clause extends object = Lib.ExpressionClause>(
       {withName && (
         <FieldWrapper>
           <FieldLabel htmlFor="expression-name">{t`Name`}</FieldLabel>
-          <Input
+          <TextInput
+            classNames={{
+              input: CS.textBold,
+            }}
             id="expression-name"
             data-testid="expression-name"
             type="text"
             value={name}
             placeholder={t`Something nice and descriptive`}
-            fullWidth
+            w="100%"
+            radius="md"
             onChange={event => setName(event.target.value)}
-            onKeyPress={e => {
+            onKeyDown={e => {
```

Comment:
onKeyPress is deprecated https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event